### PR TITLE
Fix vertical text alignment

### LINKED ISSUE #17: Improve font face metrics by using x/image/font instead of golang/freetype
This allows us to rework how the y component of the origin point for vertical text alignment is calculated, and remove the mapStringWithBearing function

### PR DESCRIPTION
**Pull request by gregjohnson2017**
Used opentype to get more accurate metrics to calculate vertical text alignment. Also touched up some organization things.

### CODE REVIEW

**Review by kroppt — APPROVED** [review]

**Comment by kroppt** [comment]
I worked on the changes myself, so getting a review from Trevor too would be best.

File: pkg/app/bottomBar.go [hunk 1]
```diff
@@ -61,7 +61,7 @@ func NewBottomBar(area *sdl.Rect, comms <-chan comms.Image, cfg *config.Config)
 	textColorID := gl.GetUniformLocation(textProgramID, &[]byte("text_color\x00")[0])
 
 	var texSheetWidth, texSheetHeight int32
-	fnt.Bind()
+	gl.BindTexture(gl.TEXTURE_2D, fnt.TextureID())
```
Comment (kropptrevor) [inline pkg/app/bottomBar.go:64]: Why is this changed back from Tyler's changes?

File: pkg/app/bottomBar.go [hunk 2]
```diff
@@ -185,7 +184,7 @@ func (bb *BottomBar) Render() {
 	gl.BindVertexArray(bb.textVaoID)
 	gl.EnableVertexAttribArray(0)
 	gl.EnableVertexAttribArray(1)
-	bb.font.Bind()
+	gl.BindTexture(gl.TEXTURE_2D, bb.fontInfo.TextureID())
```
Comment (kropptrevor) [inline pkg/app/bottomBar.go:187]: also here

**Review by kropptrevor — CHANGES_REQUESTED** [review]

File: pkg/app/bottomBar.go (hunk 2 above)
Comment (gregjohnson2017) [reply pkg/app/bottomBar.go:187]: It doesn't seem right to have one opengl call placed in a different file in our mess of opengl calls. We could have more OpenGL abstractions and make it all function calls so we are less prone to make mistakes, but I think for now we should keep it in one place

**Review by kropptrevor — APPROVED** [review]

**Pull request merged**